utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -691,6 +706,77 @@ int main(int argc, char **argv)
 
         break;
     }
+    case 82:
+        {
+            test_case_name = "ricap";
+            Rpp32u initialPermuteArray[images];
+            Rpp32u permutedArray[images * 4];
+            Rpp32u cropRegion1[4] ;
+            Rpp32u cropRegion2[4] ;
+            Rpp32u cropRegion3[4] ;
+            Rpp32u cropRegion4[4] ;
+            for (uint i = 0; i < images; i++ )
+            {
+                initialPermuteArray[i] = i;
+            }
+            randomize(initialPermuteArray, images);
+            memcpy(permutedArray, initialPermuteArray, images * sizeof(Rpp32u));
+            randomize(initialPermuteArray, images);
+            memcpy(permutedArray + images, initialPermuteArray, images * sizeof(Rpp32u));
+            randomize(initialPermuteArray, images);
+            memcpy(permutedArray + (images * 2), initialPermuteArray, images * sizeof(Rpp32u));
+            randomize(initialPermuteArray, images);
+            memcpy(permutedArray + (images * 3), initialPermuteArray, images * sizeof(Rpp32u));
+
+            RpptROI *roiPtrInputCropRegion = (RpptROI *) calloc(4, sizeof(RpptROI));
+            // xywhROI override sample
+            roiPtrInputCropRegion[0].xywhROI.xy.x = 3;
+            roiPtrInputCropRegion[0].xywhROI.xy.y = 17;
+            roiPtrInputCropRegion[0].xywhROI.roiWidth = 250;
+            roiPtrInputCropRegion[0].xywhROI.roiHeight = 238;
```

Comment:
Yes, this accepts only 300x300 Images as inputs.

I have made this generaliztion in just the Tensor_host_pk3.cpp inside rpp-unittests.
The script runs fine for input size 300x330 , 256x256
But in case of a 224x224, I am getting seg fault
Hence i have not replicated this in other test suites as of now